doc/nrf/drivers/wifi/native/nrf70.rst

```diff
@@ -87,14 +87,16 @@ Devicetree Specification configuration
 ======================================
 
 The maximum transmit power achieved on a nRF70 Series device-based product depends on the frequency band and operating channel.
-This varies from chip to chip as well as over different :term:`Printed Circuit Board (PCB)` designs.
+This varies over different :term:`Printed Circuit Board (PCB)` designs.
 
 Multiple calibrations and checks are implemented to ensure consistency across channels and devices.
 However, these values have a dependency on PCB design, which may result in Error Vector Magnitude (EVM) and spectral mask failures.
 To avoid this problem, you can specify the power ceiling at which the EVM and spectral mask are met for a given PCB design.
 Additionally, build-time parameters are made available to drivers through the DTS overlay file.
 
-The following code snippet shows an example of the DTS overlay file:
+The following code snippet shows an example of the DTS overlay file. Note that the numbers filled in the following example do not represent any particular PCB design or package type.
+These are just example numbers. Customer should fill these values based on measurements obtained from transmitter testing on their PCB design. These numbers are represented in 0.25 dB steps.
+If 15 dBm needs to be configured then value 60 (0x3c in hex) should be configured.
```

Comment:
I am suggesting a little re-wording below. Let's review and update -

The following code block shows an example of the DTS overlay file. The entries shown are only to serve as examples and don’t represent values for any particular PCB design or package type.
Customers are expected to fill in the appropriate values based on measurements obtained from transmitter testing on their custom product. The values are represented in 0.25 dB steps. For example, if 15 dBm needs to be configured then value 60 or 0x3c (hex) should be used.